docs/content/en/docs/how-tos/debug/_index.md

```diff
@@ -14,23 +14,24 @@ This functionality is in an alpha state and may change without warning.
 ## Debugging with Skaffold
 
 `skaffold debug` acts like `skaffold dev`, but it configures containers in pods
- for debugging as required for each container's runtime technology.
+for debugging as required for each container's runtime technology.
 The associated debugging ports are exposed and labelled and port-forwarded to the
 local machine.  Helper metadata is also added to allow IDEs to detect the debugging
 configuration parameters.
  
 ## How it works
 
 `skaffold debug` examines the built artifacts to determine the underlying runtime technology
-(currently supported: Java and NodeJS).  Any Kubernetes manifest that references these
+(currently supported: Java, NodeJS, and Python).  Any Kubernetes manifest that references these
```

Comment:
I think this could be highlighted more. May with some kind of heading scope? `####` so if can be referenced directly?

### supported runtimes/languages/whatever
- Java
- NodeJs
- Python

Reply:
I agree: it would be nice to make the a bit more specific.  I'll refactor this as part of the Go support, which builds on this.